### PR TITLE
Add version 14.0

### PR DESCRIPTION
**Pull request by leemannd**
- Copy 13.0 to 14.0
- Update with latest requirements
- update Dockerfile and travis to ensure the build

### CODE REVIEW

**Comment by yvaucher** [comment]
Please add 14.0 to `.travis.yml`

https://github.com/camptocamp/docker-odoo-project/blob/master/.travis.yml#L7

**Comment by leemannd** [comment]
@yvaucher Added but wasn't pushed.
Thanks for the comment

**Comment by leemannd** [comment]
@yvaucher Failing builds are not related to this PR. My proposition is to merge this one and create a BSRD Card to _maybe_ solve the failing builds for V10.0 and olders.

File: 14.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    libssl-dev \
+    libjpeg-dev \
+    libpng-dev \
+    zlib1g-dev \
+    git
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && sh -c /install/package_odoo_13.0.sh \
```
Comment (TDu) [inline 14.0/Dockerfile:35]: Shouldn't that script be updated for v14 ?

Comment (leemannd) [reply 14.0/Dockerfile:35]: Yes, you're right.

I was unsure there was anything else to be changed so I just kept it.
IMO i can rename it and update the build for V13. Do you agree?

File: 14.0/base_requirements.txt [hunk 2]
```diff
@@ -41,7 +41,6 @@ qrcode==6.1
 reportlab==3.5.13
 requests==2.21.0
 zeep==3.2.0
-python-stdnum==1.8
```
Comment (leemannd) [inline 14.0/base_requirements.txt]: This is already present below

File: 14.0/Dockerfile (hunk 1 above)
Comment (vrenaville) [reply 14.0/Dockerfile:35]: I think it's good to rename it to not confuse.
We can just copy it, I have use the image 13 to build v14 and it seems to work well.

**Review by vrenaville — APPROVED** [review]
just rename the script for fit the version number

File: 14.0/Dockerfile (hunk 1 above)
Comment (leemannd) [reply 14.0/Dockerfile]: @vrenaville @TDu I changed the name of the script according to your reviews and what has been previously done.

**Review by p-tombez — APPROVED** [review]
LG, I'm gonna check why the builds are failing for versions <=10

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**